### PR TITLE
Update basic.md

### PR DESCRIPTION
**Pull request by Jellower**
对许多翻译腔进行了修改,补充了ul在嵌套列表中的效果.

### CODE REVIEW

File: src/code/website/html/intro/basic.md [hunk 1]
```diff
@@ -45,32 +45,32 @@ tag:
...
 
-- 在可用的六个标题级别中，您应该旨在每页使用不超过三个，除非您认为有必要使用更多。具有许多级别的文档(即，较深的标题层次结构)变得难以操作并且难以导航。在这种情况下，如果可能，建议将内容分散在多个页面上。
+- 在可用的六个标题级别中，您应该旨在每页使用不超过三个，除非您认为有必要使用更多。具有许多级别的文档(即，较深的标题层次结构)是难以操作和导航的。在这种情况下，如果可能，建议将内容分散在多个页面上。
 
 :::
 
 ::: tip 为什么需要结构化
 
-这是因为没有元素给内容结构，所以浏览器不知道什么是标题，什么是段落。此外:
+因为如果没有元素给出内容结构，浏览器将不知道什么是标题，什么是段落。此外:
```
Comment (Mister-Hope) [inline src/code/website/html/intro/basic.md]:
```suggestion
因为如果元素没有给出内容结构，浏览器将不知道什么是标题，什么是段落。此外:
```

File: src/code/website/html/intro/basic.md [hunk 2]
```diff
@@ -45,32 +45,32 @@ tag:
...
 
-- 用户在阅读网页时，往往会快速浏览以查找相关内容，经常只是阅读开头的标题(我们通常在一个网页上会花费很少的时间)。如果用户不能在几秒内看到一些有用的内容，他们很可能会感到沮丧并离开。
-- 对您的网页建立索引的搜索引擎将标题的内容视为影响网页搜索排名的重要关键字。没有标题，您的网页在 SEO(搜索引擎优化)方面效果不佳。
+- 用户在阅读网页时，往往会快速浏览以查找相关内容，通常只是阅读开头的标题(我们通常在一个网页上会花费很少的时间)。如果用户不能在几秒内看到一些有用的内容，他们很可能会感到沮丧并离开。
```
Comment (Mister-Hope) [inline src/code/website/html/intro/basic.md]:
```suggestion
- 用户在阅读网页时，往往会快速浏览以查找相关内容，通常只是阅读开头的标题 (我们通常在一个网页上会花费很少的时间)。如果用户不能在几秒内看到一些有用的内容，他们很可能会感到沮丧并离开。
```

File: src/code/website/html/intro/basic.md [hunk 3]
```diff
@@ -45,32 +45,32 @@ tag:
...
+- 对您的网页建立索引的搜索引擎会将标题内容视为影响网页搜索排名的重要关键字。没有标题，您的网页在 SEO(搜索引擎优化)方面效果不佳。
```
Comment (Mister-Hope) [inline src/code/website/html/intro/basic.md]:
```suggestion
- 对您的网页建立索引的搜索引擎会将标题内容视为影响网页搜索排名的重要关键字。标题的缺失意味着您的网页在 SEO (搜索引擎优化) 方面效果不佳。
```

File: src/code/website/html/intro/basic.md [hunk 4]
```diff
@@ -45,32 +45,32 @@ tag:
...
+我们在许多地方需要依赖语义学。我们依靠以前的经验就知道日常事物都代表什么；当我们看到什么，我们就会知道它代表什么。举个例子, 我们都知道红色交通灯表示“停止”，绿色交通灯表示”通行“。
 
 同样的道理，我们需要确保使用了正确的元素来给予内容正确的意思、作用以及外形。在这里，`<h1>` 元素也是一个语义元素，它给出了包裹在您的页面上用来表示顶级标题的角色(或意义)的文本。
 
 ```html
 <h1>这是一个顶级标题</h1>
 ```
 
-一般来说，浏览器会给它一个更大的字形来让它看上去像个标题。虽然您可以使用 CSS 让它变成任何您想要的样式。更重要的是，它的语义值将以多种方式被使用，比如通过搜索引擎和屏幕阅读器(上文提到过的)。
+一般来说，浏览器会给它一个更大的字形来让它看上去像个标题。虽然您可以使用 CSS 让它变成任何您想要的样式。更重要的是，它的语义值将以多种方式被使用，比如通过搜索引擎和屏幕阅读器(正如上文提到过的)。
```
Comment (Mister-Hope) [inline src/code/website/html/intro/basic.md]:
```suggestion
一般来说，浏览器会给它一个更大的字形来让它看上去像个标题。虽然您可以使用 CSS 让它变成任何您想要的样式。更重要的是，它的语义值将以多种方式被使用，如被搜索引擎和屏幕阅读器利用 (正如上文提到过的)。
```

File: src/code/website/html/intro/basic.md [hunk 5]
```diff
@@ -82,17 +82,17 @@ tag:
...
 
 ## 列表
 
-列表在生活中随处可见，一共有三种不同类型的列表。
+列表在生活中随处可见，共有三种不同类型的列表。
 
 ### 无序 Unordered
 
-无序的列表被用来标记每个项目。在这里，项目的顺序并不重要。
+无序的列表可用来标记每个项目。在这里，项目的顺序并不重要。
```
Comment (Mister-Hope) [inline src/code/website/html/intro/basic.md]:
```suggestion
无序的列表中项目的顺序并不重要。
```

File: src/code/website/html/intro/basic.md [hunk 6]
```diff
@@ -118,7 +118,7 @@ tag:
 
 ### 有序 Ordered
 
-有序的列表是根据项目的顺序列出来的——让我们以一组方向为例:
+有序的列表将根据项目的顺序列出每一个项目——让我们以一组方向(指路)为例:
```
Comment (Mister-Hope) [inline src/code/website/html/intro/basic.md]:
```suggestion
有序的列表将根据项目的顺序列出每一个项目:
```

File: src/code/website/html/intro/basic.md [hunk 7]
```diff
@@ -150,7 +150,7 @@ tag:
 
 ### 嵌套列表 Nesting lists
 
-请务必注意正确嵌套列表，不要忘记父元素 `<ul>`
+请务必注意正确嵌套列表，不要忘记父元素 `<ul>`，每使用它一次将会创建出一次下一级。
```
Comment (Mister-Hope) [inline src/code/website/html/intro/basic.md]: Revert this change

File: src/code/website/html/intro/basic.md [hunk 8]
```diff
@@ -185,15 +185,15 @@ tag:
 
 ### 强调
 
-当我们想要在口语中添加强调，我们重读某些词，以便隐含的说出我们想要说的意思。类似的，在写作中，我们通过将文字写成斜体来强调它。比如，接下来的两个句子就有不同的含义.
+当我们想要在口语中添加强调，我们重读某些词，以便隐含地说出我们想要说的意思。类似的，在写作中，我们通过将文字写成斜体来强调它。比如，接下来的两个句子就有不同的含义.
```
Comment (Mister-Hope) [inline src/code/website/html/intro/basic.md]:
```suggestion
当我们想要在口语中添加强调，我们会通过重读某些词来表达我们想要说的意思。类似的，在写作中，我们通过将文字写成斜体来强调它。比如，接下来的两个句子就有不同的含义:
```

**Review by Mister-Hope — CHANGES_REQUESTED** [review]
One place needs to be revert

File: src/code/website/html/intro/basic.md (hunk 7 above)
Comment (Jellower) [reply src/code/website/html/intro/basic.md]: OK
